src/azure-cli/azure/cli/command_modules/role/_msgrpah/_graph_client.py

```diff
@@ -95,7 +95,7 @@ def application_delete(self, id):
         result = self._send("DELETE", "/applications/{id}".format(id=id))
         return result
 
-    def application_patch(self, id, body):
+    def application_update(self, id, body):
```

Comment:
AD Graph SDK uses verb "patch", instead of "update": `azure.graphrbac.operations.applications_operations.ApplicationsOperations.patch`. I guess it is used to differentiate between `update_application` of generic update.

We switch to use "update" to align with other update operations (such as `azure.graphrbac.operations.users_operations.UsersOperations.update`) and the API document https://docs.microsoft.com/en-us/graph/api/application-update.